src/schemathesis/_hypothesis.py

```diff
@@ -33,15 +36,22 @@ def create_test(
 ) -> Callable:
     """Create a Hypothesis test."""
     hook_dispatcher = getattr(test, "_schemathesis_hooks", None)
-    strategy = endpoint.as_strategy(hooks=hook_dispatcher)
+    from .runner.impl.core import Feedback  # pylint: disable=import-outside-toplevel
```

Comment:
Move this import to the top of the file, once the `Feedback` class will be moved